### ISSUE TITLE
[Feature Request] Microblogging support features

### ISSUE THREAD

**Issue by symbolisch1234** [GetPublii/Publii#1039]
- Currently, each post needs a post title, you can not just upload a picture and leave it at that
- Then a setting "disable post title" would be nice, so you just write post content (i think the other way around doesn't work, as a post title can't include a link or an image)
- the option to disable Theme Settings > Basic Settings > Excerpt length and just show posts in full
- (very optional) a setting to restrict post length to a user-defined length
- (very optional) auto-detection of hashtags and converting them to publii-tags

I think that is mostly it. Thanks for your consideration.

**Comment by candideu** [comment]
> Currently, each post needs a post title, you can not just upload a picture and leave it at that

That's true, **but** you *can* make it so your title doesn't appear in the page output. You just need to delete the {{title}} tag from your `post.hbs` or index, or wherever it is that you want to remove the title. You'll still need to provide a title in the post Editor, but it won't appear in your output.

See this forum topic: https://forum.getpublii.com/topic/editing-html-code-of-title-in-wysiwyg-post-editor/

> the option to disable Theme Settings > Basic Settings > Excerpt length and just show posts in full

This article might help: https://getpublii.com/dev/excerpts-with-variable-length/

Side note: Have you looked into [Mastodon](https://github.com/mastodon/mastodon)? It's made specifically for microblogging.

**Comment by symbolisch1234** [comment]
thanks for your reply. very helpful.

and yes, i looked at mastodon, it has other problems. :-)

**Comment by candideu** [comment]
> thanks for your reply. very helpful.
> 
> and yes, i looked at mastodon, it has other problems. :-)

No worries! I'm curious: what are the problems?

**Comment by symbolisch1234** [comment]
for mastodon i need an html server with node.js and ruby and yarn and PostgreSQL installed. 
for publii i need an html server.

that is the main problem. there are other, smaller ones.
